packages/cli/cypressIntegration.config.ts

```diff
@@ -0,0 +1,48 @@
+import { defineConfig } from "cypress";
+import cliPrisma from "./prisma";
+import webPrisma from "../../packages/web/prisma";
+import type { PrismaTableName } from "../../testUtilIntegration";
+
+const resetDb = async () => {
+  await truncateDatabases();
+  return null;
+};
+
+export default defineConfig({
+  e2e: {
+    specPattern: "**/*.cy.integration.{js,jsx,ts,tsx}",
+    defaultCommandTimeout: 10000,
+    baseUrl: "http://localhost:3883",
+    setupNodeEvents(on, _config) {
+      on("task", {
+        "db:reset": resetDb,
+      });
+    },
+  },
+});
+
+// copied from testUtilIntegration.ts
```

Comment:
for some reason you cannot import these modules from "../../testUtilIntegration", it will cause cypress to throw an error that the config file is invalid.  i think because ../.. is outside of packages/cli 🤷

Reply:
```$ /Users/alexfarrill/projects/mailing/node_modules/.bin/cypress run -C cypressIntegration.config.ts
Your configFile is invalid: /Users/alexfarrill/projects/mailing/packages/cli/cypressIntegration.config.ts

It threw an error when required, check the stack trace below:

Error: Cannot find module 'file:///Users/alexfarrill/projects/mailing/packages/cli/cypressIntegration.config.ts'
Require stack:
- /Users/alexfarrill/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_require_async_child.js
- /Users/alexfarrill/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/require_async_child.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Function.Module._resolveFilename (/Users/alexfarrill/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/node_modules/tsconfig-paths/lib/register.js:75:40)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/Users/alexfarrill/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at /Users/alexfarrill/Library/Caches/Cypress/10.9.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_require_async_child.js:106:14
    at processTicksAndRejections (node:internal/process/task_queues:96:5)```

Reply:
however it is fine to import a type from there (makes sense since only tsc is involved and cypress can ignore)
also ok to import `import webPrisma from "../../packages/web/prisma";`  (maybe because it's a sibling directory, not a parent?)